test/prefer-native-test.js

```diff
@@ -29,21 +30,21 @@ require('fetch-test');
       output = helpers.createBuilder(subject);
     });
 
-    afterEach(async function() {
-      await output.dispose();
-    });
+    afterEach(co.wrap(function* () {
+      yield output.dispose();
+    }));
 
-    it('preferNative is built into vendor file', async function() {
-      await output.build();
+    it('preferNative is built into vendor file', co.wrap(function*() {
+      yield output.build();
       let files = output.read();
       expect(files).to.have.all.keys('ember-fetch.js');
       expect(files['ember-fetch.js']).to.include(`var preferNative = ${preferNative}`);
-    });
+    }));
 
     it(`${
       preferNative ? 'Prefers' : "Doesn't prefer"
-    } native fetch as specified`, async function() {
-      await output.build();
+    } native fetch as specified`, co.wrap(function*() {
```

Comment:
Why use co instead of async?

Reply:
Because co is supported by Node 6 and async is not